docs/getting-started.md

```diff
@@ -11,7 +11,7 @@ Astar is the largest smart contract platform in the Polkadot ecosystem, supporti
 - [**Quick Start**](quickstart) gives you the resources you need to get started.
 testing, deploying, and interacting with smart contracts on the network.
 - [**EVM**](evm) dives deeper into EVM on Astar and explains how EVM contracts can interact with other non-EVM modules through [precompiles](evm/precompiles).
-- [**Wasm**](wasm) provides an [overview of the Wasm stack on Astar](wasm/stack/smart-contract-wasm),  Wasm [contract frameworks](wasm/stack/dsls), and their [developer tooling](wasm/sc-dev/tools/).
+- [**WASM**](wasm) provides an [overview of the WASM stack on Astar](wasm/stack/smart-contract-wasm), WASM [contract frameworks](wasm/stack/dsls), and available [developer tooling](wasm/sc-dev/tools/) including [Swanky Suite](wasm/sc-dev/tools/swanky).
```

Comment:
We have to change the name of each section in this page to make sure they align with the front page and side bar.
For example, "Quick Start" has to be "Getting Started". "EVM" has to be "Use EVM". 
I can make this change if it sounds ok for you. @andabakus

Reply:
It seems to me that getting-started.md should reflect the titles in the side bar (Docs menu). Front page is more of a selection of some of those chapters with more "promo" name. (In my opinion).
  
![image](https://user-images.githubusercontent.com/95318860/186086486-225c3457-3995-435a-ac6c-839a0f8d6fd5.png)

Maybe make a separate PR out of it @SUGURUSEO